fastlane/lib/fastlane/actions/spaceship_logs.rb

```diff
@@ -0,0 +1,132 @@
+module Fastlane
+  module Actions
+    class SpaceshipLogsAction < Action
+      def self.run(params)
+        latest = params[:latest]
+        print_contents = params[:print_contents]
+        print_paths = params[:print_paths]
+        copy_to_path = params[:copy_to_path]
+        copy_to_clipboard = params[:copy_to_clipboard]
+
+        # Get log files
+        files = Dir.glob("/tmp/spaceship*.log").sort_by { |f| File.mtime(f) }.reverse
+
+        # Filter to latest
+        if latest
+          files = [files.first]
+        end
+
+        # Print contents
+        if print_contents
+          files.each do |file|
+            data = File.read(file)
+            puts("-----------------------------------------------------------------------------------")
```

Comment:
Should this use `UI` maybe instead of `puts`? (Or doesn't it matter?)

Reply:
`UI` added that prefix to line which made it hard to read 😇

Reply:
Oh, the timestamp, right?

Reply:
Yup yup! Looked weird

Reply:
Yeah, also doesn't make sense to prefix the prefixed log.